packages/commonwealth/client/scripts/views/pages/CreateCommunity/CreateCommunity.tsx

```diff
@@ -19,6 +20,10 @@ const CreateCommunity = () => {
   const [selectedAddress, setSelectedAddress] = useState<AddressInfo>(null);
   const [createdCommunityId, setCreatedCommunityId] = useState('');
 
+  useBrowserAnalyticsTrack({
+    payload: { event: MixpanelCommunityCreationEvent.CREATE_COMMUNITY_VISITED },
+  });
```

Comment:
Since the component may render multiple times, this should be in a useEffect hook w/ empty deps array?

Reply:
Oh nvm– it's already a hook, not an individual track call.